### ISSUE TITLE
Big bold font in file description and dataset title 

### ISSUE THREAD

**Issue by lubitchv** [IQSS/dataverse#8893]
Our users would like to see file description in bigger font and bold.

![description](https://user-images.githubusercontent.com/43550154/182881497-f7856767-4d2a-4f20-b5e2-152035b4c55f.png)

They also would like to see dataset title in file above the file name.
![description_2](https://user-images.githubusercontent.com/43550154/182881516-9c950800-0fbe-4cb4-b9fa-a7091002504f.png)

**Comment by pdurbin** [comment]
For the bigger font you might be able use a custom CSS style sheet? Please see https://guides.dataverse.org/en/5.11.1/installation/config.html#custom-stylesheet

For the 10 files in the screenshot, the title of the dataset would be displayed 10 times? Above each file? Or 1 time? Above the first file?

**Comment by lubitchv** [comment]
@pdurbin Yes, for each file. Users want to have additional field (maybe called title) such that they can give more meaningful name (title) to originally uploaded file. In our case this is for librarians that curate researchers data. They do not want to change original file name but give alternative more meaninful title to it.

**Comment by pdurbin** [comment]
@lubitchv ok, where would the file title go? Can you please create a mockup for that too?

I'm sort of wondering if these changes are big enough that they should be optional. That is, if installations don't want the dataset title above every file, for example, perhaps they could opt out.
